tools/topology/sof-glk-da7219-kwd.m4

```diff
@@ -200,44 +212,70 @@ SectionGraph."pipe-sof-glk-keyword-detect" {
 # BE configurations - overrides config in ACPI if present
 #
 
+ifelse(PLATFORM, `cml',
+ifelse(SOF_ABI_VERSION_3_9_OR_GRT, `1',
```

Comment:
This conditional logic should also be in platform.m4, it looks like the only delat is SSP bclk and fsync

Reply:
@lgirdwood ok, eventually I moved all cml specific into platform.m4.